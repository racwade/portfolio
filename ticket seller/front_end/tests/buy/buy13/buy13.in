login
test_full
buy
test_sellrich
Rich
4
yes
logout