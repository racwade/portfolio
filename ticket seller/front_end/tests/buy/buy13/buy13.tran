04 Rich                test_sellrich  004 000100
00 test_full           FS 000009599