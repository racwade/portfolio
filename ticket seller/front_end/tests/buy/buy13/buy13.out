welcome to the ticket seller system
enter username:
logged in as test_full
enter seller's username:
enter event name:
enter number of tickets to buy:
total cost for 4 tickets is 400.00. confirm? (yes/no):
purchased 4 tickets to Rich from test_sellrich for 400.00
bye